src/services/apiService.ts

```diff
@@ -201,7 +201,7 @@ export class ApiService {
 
         page.value = Object.keys(tagGroups).map(x => tagGroups[x]);
         page.nextLink = pageOfApiTagResources.nextLink;
-
+        page.count =pageOfApiTagResources.count;
```

Comment:
`page.count =pageOfApiTagResources.count` - please use Shift+Alt+F for code formatting.